service-front/module/Application/view/application/authenticated/lpa/correspondent/index.twig

```diff
@@ -136,13 +136,16 @@
 
         <div class="form-group {{ topLevelError ? 'form-group-error'}}">
 
-            <fieldset id="correspondence">
-
+            <fieldset id="{{ correspondence.getAttribute('name') }}">

```

Comment:
Have to be careful here: the ID should be on the top checkbox in the set, rather than on the fieldset. This is so that the message shown in the error summary at the top of the page links to the checkbox, not the surrounding fieldset; see https://opgtransform.atlassian.net/browse/LPAL-248

Reply:
This was to bring it inline with the other fieldsets, that I came across. They were all setup in this fashion. There will only be one error on here for checkboxes & that is to make sure at least one has been selected. The ID is still the same value, it's now defined `CorrespondenceForm` instead.